modules/xmpp/JingleSessionPC.js

```diff
@@ -2030,6 +2031,49 @@ export default class JingleSessionPC extends JingleSession {
             });
     }
 
+    /**
+     * Adds a new track to the peerconnection. This method needs to be called only when a secondary JitsiLocalTrack is
+     * being added to the peerconnection for the first time.
+     *
+     * @param {JitsiLocalTrack} localTrack track to be added to the peer connection.
+     * @returns {Promise<void>} that resolves when the track is successfully added to the peerconnection, rejected
+     * otherwise.
+     */
+    addTrack(localTrack) {
+        if (!this.usesUnifiedPlan || localTrack.type !== MediaType.VIDEO) {
```

Comment:
should this also throw when the feature flag is not enabled (if called by mistake by other code)?

Reply:
It should only be called from JitsiConference.addTrack and calling it from elsewhere will result in unexpected behavior. I don't mind adding the check here.

Reply:
yes, but it will make it easier to detect when the unexpected call occurs, thanks!